modules/scheduler/module/scheduler-policy-role.tf

```diff
@@ -0,0 +1,100 @@
+data "aws_iam_policy_document" "aws_instance_scheduler_policydata" {
+  statement {
+    sid = 1
+    actions = [
+      "rds:StartDBCluster",
+      "rds:StopDBCluster",
+      "kms:Decrypt",
+      "ec2:TerminateInstances",
+      "autoscaling:ResumeProcesses",
+      "lambda:InvokeFunction",
+      "autoscaling:SuspendProcesses",
+      "rds:StopDBInstance",
+      "ec2:StopInstances",
+      "rds:StartDBInstance",
+      "logs:CreateLogStream",
+      "cloudwatch:EnableAlarmActions",
+      "ec2:StartInstances",
+      "cloudwatch:DisableAlarmActions",
+      "kms:Encrypt",
+      "autoscaling:UpdateAutoScalingGroup",
+      "autoscaling:TerminateInstanceInAutoScalingGroup",
+      "kms:CreateGrant",
+      "rds:DescribeDBClusters"
```

Comment:
Might want to remove the "ec2:TerminateInstances" permission.

Reply:
Removed the "ec2:TerminateInstances" permission